### DISCUSSION TITLE
How to share files between tart and host?

### DISCUSSION THREAD

**Discussion by onbar** [cirruslabs/tart#326] [Q&A] [answered]
Hello!
I'm just wondering... is it possible to move files from the host system to the virtual machine? Have anybody an idea?

**Comment by fkorotkov** [comment, marked as answer]
Hey @onbar,

There are couple of options:

#### Mount folders on Ventrura

If you have both Ventura host and a Ventura guest, then you can natively mount folder like that:

```bash
tart run --dir work:$PWD ventura-base
```

Then inside the VM you directory will be mounted into `/Volumes/My Shared Files/work`

#### Using Cirrus CLI

In case you want to use it for Monterey and have an ability to also script some things you can check Cirrus CLI integration. As [mentioned in ReadMe](https://github.com/cirruslabs/tart#self-hosted-ci) you can install Cirrus CLI and just run `cirrus run` which will copy over the current working directory into a running VM and will even execute scripts there. You can even [retrieve files from the VM](https://github.com/cirruslabs/tart#retrieving-artifacts-from-within-tart-vms).

#### Using `scp`

Finally, you can just use `scp` from command line and use `tart ip` for convinience:

```bash
scp file.txt admin@$(tart ip ventura-base):/Users/admin/Downloads/ 
```

**Reply by onbar** [reply]
Wow! That's great. First option works quite nice! Thanks for reply.
